$OpenBSD: patch-make_ptbuildopts_mak_in,v 1.1 2014/08/25 07:05:46 ajacoutot Exp $

INSTALL is forcibly set to ${WRKBUILD}/bin/install:
    ${WRKBUILD}/bin/install -> /usr/bin/install
which break builds of dependant ports (e.g. h323plus)

--- make/ptbuildopts.mak.in.orig	Mon Aug 25 08:47:55 2014
+++ make/ptbuildopts.mak.in	Mon Aug 25 08:48:05 2014
@@ -16,7 +16,9 @@ PTLIB_PLUGIN_DIR := @PTLIB_PLUGIN_DIR@
 AR	:= @AR@
 CC	:= @CC@
 CXX	:= @CXX@
+ifndef INSTALL
 INSTALL := @INSTALL@
+endif
 SVN     := @SVN@
 BISON   := @BISON@
 
